examples/generate_data.py

```diff
@@ -13,7 +13,7 @@
 """This is a simple script for generating data."""
 import os
 
-from openfermion.hamiltonians import make_atomic_ring
+from openfermion.chem import make_atomic_ring
```

Comment:
Might be better to just do from openfermion import <x> , as this is backwards compatible (and otherwise it appears we'll fail some of the tests here).